src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -209,7 +210,7 @@ def deactivate(
             logger.info(
                 f"Stopped Activation, Name: {instance.name}, ID: {instance.id}"
             )
-            set_activation_status(instance, ActivationStatus.STOPPED)
```

Comment:
If the deactivation triggered by scheduler is marked as `failed`, does restart will happen if policy is on_failure? do we need to mark as `failed` in this case?

Reply:
Setting FAILED status by deactivation process does not trigger auto restart. Auto restart is done by either the activation process or monitor_activation on UNRESPONSIVE activations.